.github/workflows/beekeeper.yml

```diff
@@ -86,6 +86,9 @@ jobs:
       - name: Test manifest
         id: manifest-1
         run: beekeeper check --cluster-name local-dns --checks=ci-manifest
+      - name: Test content availability
+        id: content-availability
+        run: beekeeper check --cluster-name local-dns --checks=ci-content-availability
```

Comment:
It looks like that a new beekeeper version is needed to include this check. I suppose that this is a blocker for merging this PR.